baselines/dom.generated.d.ts

```diff
@@ -12659,6 +12662,13 @@ declare var HTMLPictureElement: {
     new(): HTMLPictureElement;
 }
 
+interface parentNode {
+    children: HTMLCollection;
+    firstElementChild : Element;
+    lastElementChild : Element;
+    childElementCount: number;
+}
```

Comment:
HTML, Document and DocumentFragment interface already had been implimented NodeSelector interface. So, I think that no selector methods are required for ParentNode interface.